server/src/main/java/org/candlepin/model/Pool.java

```diff
@@ -1397,22 +1395,25 @@ public void setBranding(Set<Branding> branding) {
         this.branding = branding;
     }
 
-    @XmlTransient
+    @JsonIgnore
     public SourceSubscription getSourceSubscription() {
         return sourceSubscription;
     }
 
+    @JsonIgnore
```

Comment:
I'm not sure this mandates a comment in code, but the reason for annotating in this way are due to a number of factors:

- Going forward, this is how we should be annotating things. Jackson configuration (or behavior) can change how implied functionality works, so if we omit it, the behavior could change out from under us in the future, as has happened in a number of places in this PR.
- Throwing the annotation on the field works, but becomes inconsistent with cases where we want a getter or setter to be used, but not both. In those cases, the annotations go on the getter/setter rather than the field.
- In this particular case, the first iteration actually allowed the setter to be used (but not the getter), but in the name of sane JSON use, that was dropped and this was changed to ignore it if present.

Reply:
ack